test_schemas/en/test_custom_page_titles.json

```diff
@@ -0,0 +1,877 @@
+                                            }
+                                        ],
+                                        "type": "Radio"
+                                    }
+                                ],
+                                "id": "proxy-question",
+                                "title": {
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "person_name",
+                                            "transforms": [
+                                                {
+                                                    "arguments": {
+                                                        "delimiter": " ",
+                                                        "list_to_concatenate": {
+                                                            "identifier": ["first-name", "last-name"],
+                                                            "source": "answers"
+                                                        }
+                                                    },
+                                                    "transform": "concatenate_list"
+                                                }
+                                            ]
+                                        }
+                                    ],
+                                    "text": "Are you <em>{person_name}?</em>"
+                                },
+                                "type": "General"
+                            },
+                            "type": "Question"
+                        }
+                    ],
+                    "id": "personal-details-group",
+                    "title": "Personal Details"
+                }
+            ],
+            "id": "individual-section",
+            "repeat": {
+                "for_list": "household",
+                "title": {
+                    "placeholders": [
+                        {
+                            "placeholder": "person_name",
+                            "transforms": [
+                                {
+                                    "arguments": {
+                                        "delimiter": " ",
+                                        "list_to_concatenate": {
+                                            "identifier": ["first-name", "last-name"],
+                                            "source": "answers"
+                                        }
+                                    },
+                                    "transform": "concatenate_list"
+                                }
+                            ]
+                        }
+                    ],
+                    "text": "{person_name}"
+                }
+            }
+        },
+        {
+            "id": "question-section",
```

Comment:
Why is this in another section and not the individual section? As it stands, the hub is now missing a title:
![image](https://user-images.githubusercontent.com/1754822/92923864-0f44dc00-f430-11ea-935d-0a9d681d6332.png)

Reply:
fair point, it wasn't testing any of the repeating functionality, so I'd put it inside it's own section, though it shouldn't matter where it goes as it's isolating the question handler in the tests anyway. Chagned in 6d06416

Reply:
Done in 76c60ef3 and 6d0641